docs/api/components/schemas.yaml

```diff
@@ -1858,21 +1858,21 @@ Database:
               type: array
               items:
                 type: number
-              example: [1234, 5678]
+                example: [1234, 5678]
             disabled_users:
-              description: List of users for which audit logging is disabled
+              description: List of users for which audit logging is disabled.
               type: array
               items:
                 type: object
                 properties:
                   domain:
-                    description: The domain of the user for which audit logging is disabled.
+                    description: The domain of the user for which audit logging is disabled. Either cbs or sgw.
```

Comment:
Should the cbs/sgw be documented as enum or is that overkill/unnecessary?  I also wonder whether end users will know what 'cbs' and sgw' mean.